src/components/cardListItems.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import Menu from '@mui/material/Menu';
+import AddIcon from "@mui/icons-material/Add";
+import { TextField } from '@mui/material';
+import CloseIcon from "@mui/icons-material/Close";
+
+export default function WorkSpace() {
+  const [state, setState] = React.useState<null | HTMLElement>(null);
```

Comment:
instead of React.useState you can use import { useState } from "react";

and use useState

Reply:
Resolved